R-package/R/lgb.Booster.R

```diff
@@ -424,7 +424,7 @@ Booster <- R6::R6Class(
     },
 
     # Save model
-    save_model = function(filename, num_iteration = NULL) {
+    save_model = function(filename, num_iteration = NULL, feature_importance_type = 0) {
```

Comment:
the R linter we use requires explicitly differentiating between integers and doubles.
```suggestion
    save_model = function(filename, num_iteration = NULL, feature_importance_type = 0L) {
```